haiku-data.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
No I wrote it myself but I can look into building a script or finding one online if we need it.